### ISSUE TITLE
If the edge servers scale-in in the Auto Scaling structure, forwarding the player to a server that works seamlessly

### ISSUE THREAD

**Issue by muratugureminoglu** [ant-media/Ant-Media-Server#4778]
### Short description
There are two edge instances running on the load balancer. Watching the broadcast in one of the edge instances and the instance scale-in and the broadcast is interrupted.

1. Create a cluster using the CloudFormation Template.
2. Set desired capacity 2 for Edge configuration in Autoscale.
3. Send the stream to the origin instance
4. Watch the broadcast from edge servers. (make sure to watch on both servers)
5. Set the Desired capacity to 1 in Autoscale (Edge)
6. Check if still watch

### Expected behavior
The broadcast must continue over the edge that works seamlessly.

### Actual behavior
In the case of scale-in, the broadcast is interrupted.

**Comment by yashtandon113** [comment]
I was able to reproduce the issue on a k8s cluster. Let me explain my test scenario in the following steps:

- I created a Kubernetes cluster with one Origin, two Edge pods (Edge pod A and Edge pod B) and ingress.
- Published the stream on a Origin server, and played stream from each Edge pods using a load balancer.
- In between, reduce the number of Edge pods from two to one. After scaling down, Edge pod A was terminated in my test.

As a result, the stream that was playing via Edge pod A was interrupted and did not switch to the running Edge pod B.
